ord_schema/validations.py

```diff
@@ -562,14 +640,21 @@ def validate_reaction_outcome(message):
                         f'Undefined analysis key {key} '
                         'in ReactionProduct', ValidationError)
     if not message.products and not message.HasField('conversion'):
-        # TODO(kearnes): Should products and conversion be mutually exclusive?
         warnings.warn(
             'No products or conversion are specified for reaction;'
-            ' at least one must be specified', ValidationError)
+            ' this is permissible only for multistep reactions',
+            ValidationWarning)
```

Comment:
Note: while I changed this to a warning, we _don't_ check whether this reaction ID is used in a CrudeComponent of another reaction in the dataset. I felt like that was an unnecessarily complicated check

Reply:
Maybe let's add a TODO here? I think these kinds of "there should be a link" checks will be valuable just for hardening the submission pipeline. This might be better placed as part of `validate_dataset`.